webrtc.html

```diff
@@ -5159,7 +5159,7 @@ <h2>Methods</h2>
               non-null value, as defined in <span data-jsep=
               "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>.</p>
               <p>The <code>sendEncodings</code> argument can be used to
-              specify the number of offered simulcast encodings, and
+              specify the number of offered simulcast encodings for video, and
```

Comment:
So outside of WebRTC, we o simulcast for audio too. Why would be limit it to video ?

Reply:
@fluffy Because of costs of implementing, testing and supporting any feature? Not to mention that features without strong use-cases driving them tend to end up broken anyway. I think the question must be flipped around: Why wouldn't simulcast be limited to video?

Reply:
I'm fine with it being optional for browser to implement, but I don't think it should be forbidden. Many people in Africa are working across satellite links and they often can't use anything except very low bit rate audio while people on the thick pipes want high bandwidth. If you want to do this in a way where the media is encrypted, some clients need to send a high and low bandwidth version of the audio. Then the cloud can forward the appropriate bandwidth one to people that are listening.